Microsoft.Toolkit.Uwp.UI.Controls/Menu/MenuItem.cs

```diff
@@ -130,6 +130,11 @@ protected override void OnApplyTemplate()
                 }
             }
 
+            if (_parentMenu.CurrentFlyoutPlacement.HasValue)
```

Comment:
I know it's probably a weird scenario, but could `_parentMenu` be null?

Reply:
We should still not throw an exception even if it's used outside of a menu. Could you add the null check?

Reply:
Done